logging/README.md

```diff
@@ -3,14 +3,27 @@
 ## Kibana
 Kibana connects to Elasticsearch. Make sure you have a domain like `kibana.sandbox.xyz.net` pointing to your internal load balancer included in [global configmap](../mosip/global_configmap.yaml.sample).
 
-## Install Elasticsearch, Kibana and Istio addons
+## Install Elasticsearch, Kibana, Istio addons and logging operator.
 ```sh 
 ./install.sh
 ```
+Note: Before running the `install.sh` script please make sure the `values.yaml` is updated. 
 
-## Install Rancher FluentD system
-1. Install Logging from Apps and marketplace within the Rancher UI.
-1. Select Chart Version `103.1.1+up4.4.0` from Rancher console -> _Apps & Marketplaces_.
+## Install logging operator. 
+There are two ways to install monitoring app as given below:
```

Comment:
There are two ways to install **logging** app as given below:

Reply:
updated